### ISSUE TITLE
WOODPECKER_ADMIN should be enforced

### ISSUE THREAD

**Issue by xoxys** [woodpecker-ci/woodpecker#2004]
### Component

server

### Describe the bug

If an existing none-admin user is added to `WOODPECKER_ADMIN` Woodpecker will not grant admin privileges to this user. The expectation would be to always enforce the settings from the deployment. 

### System Info

```shell
{"source":"https://github.com/woodpecker-ci/woodpecker","version":"next-37fcc263"}
```

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/woodpecker-ci/woodpecker/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://woodpecker-ci.org/docs/intro).
- [X] Check that there isn't [already an issue](https://github.com/woodpecker-ci/woodpecker/issues) that reports the same bug to avoid creating a duplicate.
- [X] Checked that the bug isn't fixed in the `next` version already [https://woodpecker-ci.org/faq#which-version-of-woodpecker-should-i-use]
- [X] Check that this is a concrete bug. For Q&A join our [Discord Chat Server](https://discord.gg/fcMQqSMXJy) or the [Matrix room](https://matrix.to/#/#woodpecker:matrix.org).

**Comment by mzampetakis** [comment]
I could not reproduce this issue. Here are the steps followed using the `37fcc2633086e62c347119c4589b5db9dd53fd96` commit:

1. Added forge's details (bitbucket) and the following settings in .env (`my_user` is the user name within the forge): 
```
WOODPECKER_OPEN=true
WOODPECKER_ADMIN=rand_user
```
2. Started local woodpecker env and checked DB:
```
user_id|forge_remote_id|user_login|user_admin|
3|0|my_user|0|
```
3. Stoped server and started again with new config:
```
WOODPECKER_OPEN=true
WOODPECKER_ADMIN=my_user
```
4. Logged in again and checked DB:
```
user_id|forge_remote_id|user_login|user_admin|
3|0|my_user|1|
```
and also I was able to admin admin settings through woodpecker UI.

I am not sure if I am missing something here...  🤔
